joss_paper/paper.md

```diff
@@ -0,0 +1,289 @@
+  - name: Tiago M. D. Pereira
+    orcid: 0000-0003-4747-4329
+    affiliation: 
+      - "Rosseland Centre for Solar Physics, University of Oslo, P.O. Box 1029 Blindern, NO-0315 Oslo, Norway"
+      - "Institute of Theoretical Astrophysics, University of Oslo, P.O. Box 1029 Blindern, NO-0315 Oslo, Norway"
+  - name: Yudhik Agrawal
+    orcid: 0000-0003-3827-6857
+    affiliation: 'International Institute of Information Technology, Hyderabad 500 032, India'
+  - name: jamescalixto
+  - name: yasintoda
+  - name: Sophie A. Murray
+    orcid: 0000-0002-9378-5315
+    affiliation: Trinity College Dublin, Ireland
+
+affiliations: []
+date: 3 October 2019
+bibliography: paper.bib
+aas-doi: "Add DOI HERE"
+aas-journal: "The Astrophysical Journal"
+---
+
+# Summary
+
+The Sun, our nearest star, is a local laboratory for studying universal physical processes. Solar physics as a discipline includes studying the Sun both as a star and as the primary driver of space weather throughout the heliosphere. Due to the Sun's proximity, the temporal and spatial resolution of solar observations are orders of magnitude larger than those of other stars. This leads to significant differences in the data-analysis software needs of solar physicists compared with astrophysicists.
+
+The `sunpy` Python package is a community-developed, free, and open-source solar data analysis environment for Python that provides a comprehensive set of tools for performing common tasks in solar data analysis. It is managed by the SunPy Project, an organization that facilitates and promotes the use of open development and open source packages like `sunpy` through community engagement and tools such as [GitHub](https://github.com/sunpy/sunpy/), [mailing lists](https://groups.google.com/forum/#!forum/sunpy), and [matrix](https://matrix.to/#/#sunpy:openastronomy.org).
+
+The four most significant subpackages of `sunpy` are described below.
+
+The `sunpy.net` subpackage provides a unified interface that simplifies and homogenizes search and retrieval by querying and downloading data from many solar data sources, irrespective of the underlying data-source client. It currently supports sourcing data from 18 different space- and ground-based solar observatories.
+
+The `sunpy.map` and `sunpy.timeseries` subpackages provide core data types (`Map` and `TimeSeries`, respectively) that are designed to provide a general, standard, and consistent interface for loading and representing solar data across different instruments and missions. These classes load data which conform to solar physics standards and conventions such as FITS [@wells_fits_1981], FITS World Coordinate Systems (WCS) [@fits_wcs], and solar-specific FITS headers [@thompson_coordinates_2006], while allowing customization to account for differences in specific instruments. Visualization methods are also provided to inspect and plot those data. Example visulizations of both `TimeSeries` and `Map` are shown in Figure 1.
```

Comment:
The reference `@thompson_coordinates_2006` is showing up in [this rendered PDF](https://github.com/openjournals/joss-reviews/files/3829309/10.21105.joss.01832.pdf) as `???`. Looks like it is `coordinate` (singular) in the BiBTeX entry, not plural.

Reply:
👍 Fixed by https://github.com/sunpy/sunpy/commit/6188dad4528161f7096954cd569f6d9399d8b697